code/game/turfs/simulated.dm

```diff
@@ -16,6 +16,10 @@
 /turf/simulated/proc/burn_tile()
 	return
 
+/turf/simulated/cleaning_act(mob/user, atom/cleaner, cleanspeed = 50, text_verb = "clean", text_description = " with [cleaner].")
```

Comment:
any reason for the space in the text_description parameter here?

Reply:
Yes, [text_description] is inserted immediately after [name]. This is necessary in case text_description is ".", in which case you'd want the period immediately after the last word in the sentence. Because of this, in cases where text_description is _not_ just a period, a space is inserted at the beginning.